### PR TITLE
Update docs from testing

### PR DESCRIPTION
**Pull request by benjamin-heasly**
Greetings! My name is Ben Heasly, I'm a developer working with the Nassar lab at Brown Neuroscience to create some online tasks. I'm getting myself up to speed with Honeycomb, following the Quick Start docs here (which are great).

I just made a [Honeycomb PR](https://github.com/brown-ccv/honeycomb/pull/96) with some minor changes that I needed to get myself up and running.  I also want to offer up some small docs changes that I found here.

@mirestrepo would you be a good reviewer for this? And/or could you suggest some reviewers?

Thanks 1e6!

### CODE REVIEW

File: docs/quick_start.md [hunk 1]
```diff
@@ -164,3 +164,31 @@ To merge:
 ```
 git commit -a -m "merge honeycomb latest"
 ```
+
+### 8. Run automated tests
```
Comment (mirestrepo) [inline docs/quick_start.md:168]: Do we have tests?

**Review by mirestrepo — COMMENTED** [review]
Hi Ben! This is great! Thank you! 

I only have one question. Do we have tests? I don't see any on the Honeycomb repo....

**Comment by mirestrepo** [comment]
I've also add @hollandjg in case he would like to add anything

File: docs/quick_start.md (hunk 1 above)
Comment (benjamin-heasly) [reply docs/quick_start.md:168]: Oh, I did see some running.  I think they're the ones here:
https://github.com/brown-ccv/honeycomb/blob/main/src/__tests__/utils.test.js

So not a lot of tests, but I thought it seemed right to run them.

Comment (mirestrepo) [reply docs/quick_start.md:168]: Ha! I don't know why I missed them. Looks great!

**Review by mirestrepo — APPROVED** [review]

**Pull request merged**